samples/storage_csm_powerstore_v250.yaml

```diff
@@ -65,6 +65,12 @@ spec:
         # Default value: false
         - name: X_CSI_HEALTH_MONITOR_ENABLED
           value: "false"
+        # X_CSI_HEALTH_MONITOR_ENABLED: allows to specify additional entries for hostAccess of NFS volumes. Both single IP address and subnet are valid entries.
+        # Install the 'external-health-monitor' sidecar accordingly.
```

Comment:
Please remove this line as it is not related to this environment variable

Reply:
take care